### DISCUSSION TITLE
prefixing support for utility class names

### DISCUSSION THREAD

**Discussion by zahmo** [saadeghi/daisyui#2893] [Ideas / feature requests]
Hi, 
currently only prefixing of DaisyUI class names are supported, however sometimes it seems that other third party class names
use the !important rule, so for example DaisyUIs 'bg-primary' will have no effect (even if its loaded after). So one suggestion is to implement prefixing for these class names as well.

Thanks

**Comment by saadeghi** [comment]
It's not possible. `primary` is a color from daisyUI but the way it works (because of Tailwind CSS API) I should add the colors to Tailwind CSS and Tailwind CSS generates the `bg-primary` class. Basically `bg-primary` class is a Tailwind CSS class. daisyUI doesn't send it to Tailwind as a class name. It just sends the color name.

Class names like `btn` are being sent to Tailwind CSS directly so daisyUI is able to process them before sending them to Tailwind.

Let me know if you have any questions

**Reply by zahmo** [reply]
Thanks for the prompt reply, one workaround would be to use both the tailwind and daisyui prefix options, however this will create class names like ```tw-dui-btn-seconadary``` for the daisyui controlled names and ```tw-bg-primary``` for the tailwind controlled names.
